pygmt/tests/test_datasets_earth_vertical_gravity_gradient.py

```diff
@@ -21,7 +21,7 @@ def test_earth_vertical_gravity_gradient_01d():
     npt.assert_allclose(data.lon, np.arange(-180, 181, 1))
     npt.assert_allclose(data.min(), -137.125, atol=1 / 32)
     npt.assert_allclose(data.max(), 104.59375, atol=1 / 32)
-    assert data[1, 1].isnull()
+    assert data[1, 1].isna()
```

Comment:
This is checking on an `xarray.DataArray` which uses [`isnull`](https://docs.xarray.dev/en/stable/generated/xarray.DataArray.isnull.html)
```suggestion
    assert data[1, 1].isnull()
```